media-source-respec.html

```diff
@@ -2723,10 +2726,19 @@
         </thead>
         <tbody>
           <tr>
-            <td>April 15 2016</td>
+            <td>April 28 2016</td>
             <td>
               <ul>
-                <li>Issue 17 &amp; 18 - Generalize the detach MediaSource algorithm.</li>
+                <li>Issue 54 - Clarified use of Presentation Time Stamps (PTS) for <a def-id="buffered"></a>.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <td><a href="https://rawgit.com/w3c/media-source/365acf8c3c50aa253439782894d5ee9492b81039/index.html">April 15 2016</td>
```

Comment:
This URL points to a version of the spec that is timestamped Nov 12, 2015. You probably want to instead reference https://rawgit.com/w3c/media-source/a0bf0e68c8e5e53a9450abe94b83a9c3a6f8f13e/index.html 

Please also update media-source-respec.html's publishDate and regenerate index.html.